package.json

```diff
@@ -18,7 +18,7 @@
     "dist"
   ],
   "scripts": {
-    "prepack": "nuxt-module-build build",
+    "prepack": "nuxt-module-build prepare && nuxt-module-build build",
```

Comment:
```suggestion
    "prepack": "npm run dev:prepare && nuxt-module-build build",
```

Reply:
I still don't understand why we need to prepare the playground to build the module, we can have multiple playgrounds.

I'll try to check your suggestion later, check my Vuetify module scripts: https://github.com/vuetifyjs/nuxt-module/blob/main/package.json#L45-L56 (I need to update the dependencies, using old `@nuxt/module-builder v0.8.3`).

Reply:
you don't necessarily need to prepare the playground to build the module, but the 'dev:prepare' step should be run after installation. it's necessary to ensure types are correctly generated.